dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -413,6 +413,7 @@ def load_user
       @user = user_to_view
 
       if can?(:view_as_user_for_code_review, @script_level, user_to_view, sublevel_to_view)
+        @is_code_reviewing = true
         view_options(is_code_reviewing: true)
```

Comment:
Why do we need to add another `@is_code_reviewing` constant rather than using the view option that already exists?

Reply:
Mostly, yes. There is an example here of us referencing a view option in haml https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/levels/_free_response.html.haml#L48. You can leave it, I was just curious

Reply:
And by back-end controller code path, you mean the haml file right?

Reply:
Controller logic (_controller.rb files) + haml files.